SCClassLibrary/Common/Control/Bus.sc

```diff
@@ -55,6 +55,7 @@ Bus {
 	}
 
 	setMsg { arg ... values;
+		if(index.isNil) { Error("Cannot construct a % for a % that has been freed".format(thisMethod.name, this.class.name)).throw };
 		if(this.isSettable, {
```

Comment:
Good point. What is the current preferred syntax for if statements? I don't mind making these changes.